src/models/NeverminedOptions.ts

```diff
@@ -1,39 +1,46 @@
 import { ethers } from 'ethers'
-import { ZeroAddress } from '../utils'
+import { ZeroAddress } from '../utils/ConversionTypeHelpers'
 import { LogLevel } from '../utils/Logger'
 import { AaveConfig } from './AaveConfig'
 export { LogLevel } from '../utils/Logger'
 
-export class Config {
+export class NeverminedOptions {
```

Comment:
I think this can be just an interface

Reply:
Interfaces don't allow to specify default values, and there are some attributes here that need to have some default values initialization